File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
  let lux = wallpaper[0].length;
  let luy = wallpaper.length;
  let rdx = 0;
  let rdy = 0;

  for (let i = 0; i < wallpaper.length; i++) {
    for (let j = 0; j < wallpaper[0].length; j++) {
      if (wallpaper[i][j] === "#") {
        if (lux > j) {
          lux = j;
        }

        if (luy > i) {
          luy = i;
        }

        if (rdx < j) {
          rdx = j;
        }

        if (rdy < i) {
          rdy = i;
        }
      }
    }
  }

  return [luy, lux, rdy + 1, rdx + 1];
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    lux = len(wallpaper[0])
    luy = len(wallpaper)
    rdx = 0
    rdy = 0

    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == "#":
                if lux > j:
                    lux = j
                if luy > i:
                    luy = i
                if rdx < j:
                    rdx = j
                if rdy < i:
                    rdy = i

    return [luy, lux, rdy + 1, rdx + 1]
